tests/Unit/DataStructures/Tensor/EagerMath/Test_DeterminantAndInverse.cpp

```diff
@@ -248,4 +251,98 @@ SPECTRE_TEST_CASE("Unit.DataStructures.Tensor.EagerMath.DeterminantAndInverse",
     CHECK((get<1, 0>(det_inv.second)) == DataVector({2.0, 2.0, -1.0, -0.5}));
     CHECK((get<1, 1>(det_inv.second)) == DataVector({-1.0, -3.0, 2.5, 0.5}));
   }
+
+  // Check Variables<determinant, inverse> for a Tensor<DataVector>
+  {
+    {
+      // 2D
+      using my_tnsr_type = tnsr::ij<DataVector, 2, Frame::Grid>;
+      using my_tnsr_inv_type = tnsr::IJ<DataVector, 2, Frame::Grid>;
+      using my_tnsr_det_type = Scalar<DataVector>;
+
+      struct MyDetTag : db::SimpleTag {
+        static std::string name() noexcept { return "DummyDetTag"; }
+        using type = my_tnsr_det_type;
+      };
+      struct MyInvTag : db::SimpleTag {
+        static std::string name() noexcept { return "DummyInvTag"; }
+        using type = my_tnsr_inv_type;
+      };
+
+      my_tnsr_type t{};
+      get<0, 0>(t) = DataVector({2.0, 3.0, 5.0, 1.0});
+      get<0, 1>(t) = DataVector({3.0, 5.0, 4.0, -1.0});
+      get<1, 0>(t) = DataVector({4.0, 2.0, 2.0, 1.0});
+      get<1, 1>(t) = DataVector({5.0, 3.0, 2.0, 1.0});
+      const auto det_inv = determinant_and_inverse<MyDetTag, MyInvTag>(t);
+      CHECK(get(get<MyDetTag>(det_inv)) == DataVector({-2.0, -1.0, 2.0, 2.0}));
```

Comment:
Not a change request but just an FYI: I believe you can just write `DataVector{-2.0, -1.0, 2.0, 2.0}` without the extra parens.